src/Microsoft.ML.Data/Dirty/PredictorInterfaces.cs

```diff
@@ -202,6 +202,23 @@ internal interface IFeatureContributionMapper : IPredictor
         ValueMapper<TSrc, VBuffer<float>> GetFeatureContributionMapper<TSrc, TDst>(int top, int bottom, bool normalize);
     }
 
+    /// <summary>
+    /// Allows support for feature contribution calculation.
+    /// </summary>
+    public interface IFeatureContributionMappable : IPredictor
```

Comment:
>public interface IFeatureContributionMappable : IPredictor [](start = 4, length = 58)

Incidentally I think this is a pretty good example of the encapsulating pattern.